vnet/net.go

```diff
@@ -489,9 +493,21 @@ func NewNet(config *NetConfig) *Net {
 		Flags:        net.FlagUp | net.FlagMulticast,
 	})
 
+	var staticIPs []net.IP
+	for _, ipStr := range config.StaticIPs {
+		if ip := net.ParseIP(ipStr); ip != nil {
```

Comment:
should we show a warn in case the ip cannot be parsed?

Reply:
Oh, yes I agree!  I will add a warning log before I merge!

Reply:
Oops...  NewNet() currently does not take a logger. Also, it does not return error either. :(
I believe it should return an error in this case. I am going to file an issue and work on it later as it would be a breaking change.

Reply:
perfect! thanks!

Reply:
Filed it as #34.